core/lib/basic_types/src/web3/tests.rs

```diff
@@ -83,3 +83,12 @@ fn block_can_be_deserialized() {
     let block: Block<H256> = serde_json::from_str(pre_dencun).unwrap();
     assert!(block.excess_blob_gas.is_none());
 }
+
+#[test]
+fn test_bytes_serde() {
+    use bincode::{deserialize, serialize};
```

Comment:
Bikeshedding: probably using fully qualified paths here would be more expressive.